pkg/ingress/controller/apisix_service.go

```diff
@@ -16,8 +16,8 @@ package controller
 
 import (
 	"fmt"
-	"github.com/api7/ingress-controller/pkg/ingress/apisix"
-	"github.com/golang/glog"
+	"time"
```

Comment:
The effect of this `time` was not found in this change.

Reply:
![image](https://user-images.githubusercontent.com/10428333/102193292-e16fa780-3ef6-11eb-8ab6-840a9f50b7d6.png)

It's used here.